thangs_fetcher.py

```diff
@@ -361,6 +362,9 @@ def display_search_results(self, responseData, show_summary=True):
             try:
                 thumb = self.pcoll.load(item["modelId"], image_path, 'IMAGE')
             except Exception as e:
+                # AFAIK this covers an edge case where search results have duplicate models and also duplicate model ID's.
+                # The pcoll elements need to be unique so we're adding the index to the end of the model ID.
+                thumb = self.pcoll.load(item["modelId"]+str(I), image_path, 'IMAGE')

```

Comment:
I was skeptical of this line before. The new amplitude event here confirmed it was there for a good reason so I've added the line back in.